pom.xml

```diff
@@ -45,6 +46,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
Is this dependency needed? I don't see why adding a workflow step would require the `junit` plugin.